pkg/datagatherer/k8s/generic.go

```diff
@@ -51,8 +54,17 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 // validate validates the configuration.
 func (c *Config) validate() error {
+	var errors []string
+	if len(c.ExcludeNamespaces) > 0 && len(c.IncludeNamespaces) > 0 {
+		errors = append(errors, "cannot set excluded an included namespaces")
```

Comment:
```suggestion
		errors = append(errors, "cannot set excluded and included namespaces")
```

Reply:
Fixed in d12572780845107463c22448986dd000130147e4 (manually to add commit sign-off)